CONTRIBUTING.md

```diff
@@ -62,8 +62,9 @@ of both the contributors and the project.
 
 ## Style Guide
 
-This project follows the
-[Google Python style guide](https://google.github.io/styleguide/pyguide.html).
+This project primarily follows the [PEP 8 style guide](https://www.python.org/dev/peps/pep-0008/), and 
```

Comment:
Please make this change for CONTRIBUTING.rst instead. To display external hyperlinks in restructuredText please follow the following format:  
`support Pipfiles <https://github.com/readthedocs/readthedocs.org/issues/3181>`__

Reply:
Done!